src/SmartComponents/Inventory/applications/compliance/SystemPolicyCard.js

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import { CheckCircleIcon, ExclamationCircleIcon } from '@patternfly/react-icons';
+import Truncate from 'react-truncate';
```

Comment:
Isn't this essentially the same as our long text truncate? [/src/PresentationalComponents/LongTextTooltip/LongTextTooltip.js](https://github.com/RedHatInsights/insights-frontend-components/blob/master/src/PresentationalComponents/LongTextTooltip/LongTextTooltip.js)

Reply:
This isn't a tooltip. Is that component misnamed or something?

Reply:
I think @karelhala meant to link: https://github.com/RedHatInsights/insights-frontend-components/blob/master/doc/components/truncate.md

Reply:
Is there a way to use auto length? How am I supposed to know how many characters? That depends on the width of the browser, right?

Reply:
That's true. We may want to go back and revisit that issue with the other component. This should be fine for now.